lib/utils/token/accountApi/callAccountApi.ts

```diff
@@ -0,0 +1,36 @@
+import { StorageKeys } from "../../../sessionManager";
+import { getActiveStorage, getClaim } from "../index";
+
+export type AccountApiRoute = `account_api/${string}`;
+
+export async function callAccountApi<T>(route: AccountApiRoute): Promise<T> {
+  const storage = getActiveStorage();
+  if (!storage) {
+    throw new Error("No active storage found.");
+  }
+
+  const token = await storage.getSessionItem(StorageKeys.accessToken);
+  if (!token) {
+    throw new Error("Authentication token not found.");
+  }
+
+  const domain = await getClaim("iss");
+  if (!domain?.value) {
+    throw new Error("Domain (iss claim) not found.");
+  }
+
+  const response = await fetch(`${domain.value}/${route}`, {
+    method: "GET",
```

Comment:
@DanielRivers I suspect we'll need to consider POST requests later also for updating emails / users name etc